packages/community-cli-plugin/src/commands/start/attachKeyHandlers.js

```diff
@@ -19,6 +19,18 @@ import fetch from 'node-fetch';
 
 const CTRL_C = '\u0003';
 const CTRL_D = '\u0004';
+const reloadTimeout = 700;
```

Comment:
nit: Constant naming
```suggestion
const RELOAD_TIMEOUT = 700;
```